ner_v2/detectors/numeral/number/number_detection.py

```diff
@@ -158,13 +158,16 @@ def detect_entity(self, text, **kwargs):
         for number_value_dict, original_text in zip(number_data[0], number_data[1]):
             number_value = number_value_dict[NUMBER_DETECTION_RETURN_DICT_VALUE]
             number_unit = number_value_dict[NUMBER_DETECTION_RETURN_DICT_UNIT]
-            if self.min_digit <= self._num_digits(number_value) <= self.max_digit:
-                if self.unit_type and (number_unit is None or
-                                       self.language_number_detector.units_map[number_unit].type != self.unit_type) \
-                        and not self.detect_without_unit:
-                    continue
-                validated_number.append(number_value_dict)
-                validated_number_text.append(original_text)
+            try:
+                if self.min_digit <= self._num_digits(number_value) <= self.max_digit:
+                    if self.unit_type and (number_unit is None or
+                                          self.language_number_detector.units_map[number_unit].type != self.unit_type) \
```

Comment:
E128 continuation line under-indented for visual indent
E501 line too long (120 > 119 characters)